test/unit/index-test.js

```diff
@@ -93,16 +98,19 @@ describe('index.js', function() {
       it('does not add POST endpoint to app', function() {
         expect(app.post.callCount).to.equal(0);
       });
+      it('does not GET endpoint to app', function() {
+        expect(app.get.callCount).to.equal(0);
+      });
     });
   });
 
-  describe('_doesFileExistInCurrentProject', function() {
+  describe('_doesFileExistInCurrentProjectApp', function() {
```

Comment:
No. I think I might have snuck it in the previous as well. Since `doesFileExistInCurrentProject1` was renamed to `doesFileExistInCurrentProject` -> `doesFileExistInCurrentProjectApp` but the tests weren't